docs/architecture/grpc-for-wcf-developers/metadata.md

```diff
@@ -7,5 +7,53 @@ ms.date: 09/02/2019
 
 # Metadata
 
+"Metadata" refers to additional data that may be useful while processing requests and responses but is not part of the actual application data. Metadata might include authentication tokens, request identifiers and tags for monitoring purposes, or information about the data like the number of records in a dataset.
```

Comment:
Consider comparing to WCF custom SOAP headers so that devs have an answer to "How do I map my usage of custom SOAP headers to gRPC?"